<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>88</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Save Changes?</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../lib/lib.qrc">
    <normaloff>:/synthclone/images/16x16/important.png</normaloff>:/synthclone/images/16x16/important.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" stretch="1,0">
   <item>
    <layout class="QHBoxLayout" stretch="0,1">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../lib/lib.qrc">:/synthclone/images/32x32/important.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>The current session has been modified.  Do you want to save your changes before closing the session?</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" stretch="1,0,0,0">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../lib/lib.qrc">
         <normaloff>:/synthclone/images/16x16/close.png</normaloff>:/synthclone/images/16x16/close.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="discardButton">
       <property name="text">
        <string>Discard Changes</string>
       </property>
       <property name="icon">
        <iconset resource="../lib/lib.qrc">
         <normaloff>:/synthclone/images/16x16/quit.png</normaloff>:/synthclone/images/16x16/quit.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Save Changes</string>
       </property>
       <property name="icon">
        <iconset resource="../lib/lib.qrc">
         <normaloff>:/synthclone/images/16x16/save.png</normaloff>:/synthclone/images/16x16/save.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../lib/lib.qrc"/>
 </resources>
 <connections/>
</ui>
